src/functionalTest/resources/sourceFiles/service-test.csv

```diff
@@ -0,0 +1,3 @@
+service_code,ccd_service_name,ccd_case_type
+AAA1,ccd-service1,"service1,service2"
```

Comment:
Valid file -

service_code,ccd_service_name,ccd_case_type
AAA1,ccd-service1,"service1,service2"
AAA2,ccd-service2,"service11,service12"
AAA3, ccd-service3,
AAA4,,

Reply:
ideally file wont include - AAA4,, but we should be able to handle it and ignore in case its there

Reply:
This is covered in unit test. Ideally code logic is handled in unit cases.

Reply:
This is not about code logic. This should be tested in functional Include this scenario in functional test